SConstruct

```diff
@@ -1397,7 +1394,8 @@ def pythonVersion( pythonEnv ) :
 
 	pythonExecutable = pythonEnv.subst( "$PYTHON" )
 	env = pythonEnv["ENV"].copy()
-	env[libraryPathEnvVar] = pythonEnv.subst( os.pathsep.join( pythonEnv["LIBPATH"] ) )
+	if "TEST_LIBRARY_PATH_ENV_VAR" in pythonEnv and pythonEnv["TEST_LIBRARY_PATH_ENV_VAR"] :
```

Comment:
But also, I don't think we need to switch from `libraryPathEnvVar` to `TEST_LIBRARY_PATH_ENV_VAR` here anyway do we? The latter is advertised as being used when running the tests, but we're not running any tests here.

Reply:
fixed in https://github.com/ImageEngine/cortex/pull/1181/commits/a641c41f73c4b9598b3fd81975c5f3cb5a1dadf8. My brain was crossed with the intention behind `libraryPathEnvVar` and `TEST_LIBRARY_PATH_ENV_VAR` but I get it now.

I changed the `PATH` fix to a much simpler solution affecting only `imageEnv` and `testEnv` and not meddling with the other tests and uses of `TEST_LIBRARY_PATH_ENV_VAR` which already append rather than replace the value.